packages/hoprd/src/api/v3/paths/node/peers.ts

```diff
@@ -80,7 +85,7 @@ export async function getPeers(
 
     for (const peerId of connected_peers) {
       const peerIdStr = peerId.toString()
-
+      const chainKey = await node.peerIdToChainKey(peerId)
```

Comment:
Note that `peerIdToChainKey` can fail, if we haven't indexed the KeyBinding.

Make sure that appropriate error handling is done on each call.

Reply:
Done.